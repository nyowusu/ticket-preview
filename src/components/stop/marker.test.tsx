import { shallow } from 'enzyme';
import React from 'react';

import Component from './marker';

describe('StopMarker', () => {
  it('renders', () => {
    /* const stop = {
      type: 'Feature',
      geometry: {
        type: 'Point',
        coordinates: [39.26375176777378, -6.778027365319448],
      },
      properties: {
        stop_id: 'MOROCCO05',
        stop_name: 'Morocco BRT',
        wheelchair_boarding: 0,
        zone_id: 'MOROCCO05',
        agency_key: 'Safiri',
        agency_name: 'Safiri',
      },
    }; */

    expect(
      shallow(
        <Component type="bus" onPress={() => console.log(`You clicked`)} />,
      ),
    ).toMatchSnapshot();
  });
});
