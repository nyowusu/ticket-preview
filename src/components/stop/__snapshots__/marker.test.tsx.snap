// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StopMarker renders 1`] = `
<MapMarker
  anchor={
    Object {
      "x": 0.5,
      "y": 0.5,
    }
  }
  coordinate={
    Object {
      "latitude": 0,
      "longitude": 0,
    }
  }
  height={50}
  onPress={[Function]}
  stopPropagation={false}
  style={
    Object {
      "zIndex": 1000,
    }
  }
  width={50}
>
  <StopIcon
    size={50}
    type="bus"
  />
</MapMarker>
`;
